Dockerfile.release

```diff
@@ -1,23 +1,31 @@
-FROM bitwalker/alpine-erlang:19.2.1b
+FROM phusion/baseimage
```

Comment:
Well, MongooseIM images don't have the DNS issues like the image of MongoosePush. Also I heard `alpine` is known for such issues on our `clt-swarm`. Therefore I want to use same image as MongooseIM.